Source/Fuse.Navigation/RouterPage.uno

```diff
@@ -167,58 +166,58 @@ namespace Fuse.Navigation
 					cur.SubRoute = nxt;
 					cur = nxt;
 				}
-				
+
 				r = r.SubRoute;
 			}
-			
+
 			return bas;
 		}
-		
+
 		public Route ToRoute()
 		{
-			var r = new Route( RouterPage.Path, RouterPage.Parameter, 
+			var r = new Route( RouterPage.Path, RouterPage.Parameter,
 				SubRoute != null ? SubRoute.ToRoute() : null );
 			r.RouterPage = RouterPage;
 			return r;
 		}
-		
+
 		internal RouterPageRoute Up()
 		{
 			if (SubRoute == null) return this;
 			else if (SubRoute.SubRoute == null) return new RouterPageRoute(RouterPage, null);
 			return new RouterPageRoute( RouterPage, SubRoute.Up());
 		}
-		
+
 		internal string Format()
 		{
 			var q = RouterPage.ToPathString();
 			if (SubRoute != null)
 				q += "/" + SubRoute.Format();
 			return q;
 		}
-		
+
 		internal RouterPageRoute SubDepth(int count)
 		{
 			if (count <0)
 			{
 				Fuse.Diagnostics.InternalError( "count can't be < 0", this );
 				return null;
 			}
-			
+
 			if (count == 0)
 				return this;
-				
+
 			if (SubRoute == null)
 				return null;
-				
+
 			return SubRoute.SubDepth(count-1);
 		}
-		
+
 		public RouterPageRoute Append( RouterPageRoute subRoute )
 		{
 			var sub = SubRoute == null ? subRoute : SubRoute.Append(subRoute);
 			return new RouterPageRoute(RouterPage, sub);
 		}
-		
+
```

Comment:
Looks like the changes in this file is just white-space cruft now, can we please drop it?